src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -321,14 +220,52 @@
                     <Style x:Key="ThemedSwitchModeToggleButtonStyle"
                            BasedOn="{StaticResource SwitchModeToggleButtonStyle}"
                            TargetType="ToggleButton"/>
+                    <Style x:Key="GraphControlCommandPanel" TargetType="Border">
+                        <Setter Property="Background" Value="#80000000"/>
+                        <Setter Property="CornerRadius" Value="4"/>
+                    </Style>
+                    <Style x:Key="ThemedGraphRepeatButtonStyle"
+                           BasedOn="{StaticResource GraphRepeatButtonStyle}"
+                           TargetType="RepeatButton"/>
+                    <Style x:Key="ThemedGraphButtonStyle"
+                           BasedOn="{StaticResource GraphButtonStyle}"
+                           TargetType="Button"/>
+                    <Style x:Key="ThemedGraphToggleButtonStyle"
+                           BasedOn="{StaticResource GraphToggleButtonStyle}"
+                           TargetType="ToggleButton"/>
                 </ResourceDictionary>
-                <ResourceDictionary x:Key="Dark">
+                <ResourceDictionary x:Key="Light">
                     <Style x:Key="ThemedSwitchModeToggleButtonStyle"
                            BasedOn="{StaticResource SwitchModeToggleButtonStyle}"
                            TargetType="ToggleButton"/>
+                    <Style x:Key="GraphControlCommandPanel" TargetType="Border">
+                        <Setter Property="Background" Value="{StaticResource SystemControlAcrylicElementBrush}"/>
+                        <Setter Property="BorderBrush" Value="#e0e0e0"/>
+                        <Setter Property="BorderThickness" Value="1"/>
+                        <Setter Property="CornerRadius" Value="4"/>
+                    </Style>
+                    <Style x:Key="ThemedGraphRepeatButtonStyle"
+                           BasedOn="{StaticResource GraphRepeatButtonStyle}"
+                           TargetType="RepeatButton"/>
+                    <Style x:Key="ThemedGraphButtonStyle"
+                           BasedOn="{StaticResource GraphButtonStyle}"
+                           TargetType="Button"/>
+                    <Style x:Key="ThemedGraphToggleButtonStyle"
+                           BasedOn="{StaticResource GraphToggleButtonStyle}"
+                           TargetType="ToggleButton"/>
                 </ResourceDictionary>
                 <ResourceDictionary x:Key="HighContrast">
```

Comment:
Many of the buttons are not legible in high contrast black and other high contrast modes. The background/foreground should match the CalculatorButtons on the right side

Reply:
The only issue I found was when buttons are in pointerOver and Pressed states. I will fix that!

Interesting, if you write:

```
   <ResourceDictionary.ThemeDictionaries>
                            <ResourceDictionary x:Key="Light">
                                <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="#10000000"/>
                                <SolidColorBrush x:Key="ButtonBackgroundPressed" Color="#20000000"/>
                                <SolidColorBrush x:Key="RepeatButtonBackgroundPointerOver" Color="#10000000"/>
                                <SolidColorBrush x:Key="RepeatButtonBackgroundPressed" Color="#20000000"/>
                            </ResourceDictionary>
                        </ResourceDictionary.ThemeDictionaries>
``` 
It will still use the `Light` resources for high-contrast.

To fix the issue, you need to add an empty ResourceDictionary

```
   <ResourceDictionary.ThemeDictionaries>
                            <ResourceDictionary x:Key="Light">
                                <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="#10000000"/>
                                <SolidColorBrush x:Key="ButtonBackgroundPressed" Color="#20000000"/>
                                <SolidColorBrush x:Key="RepeatButtonBackgroundPointerOver" Color="#10000000"/>
                                <SolidColorBrush x:Key="RepeatButtonBackgroundPressed" Color="#20000000"/>
                            </ResourceDictionary>
                            <ResourceDictionary x:Key="HighContrast"/>
                        </ResourceDictionary.ThemeDictionaries>
```

Reply:
About the colors: CalculatorButton doesn't match the system colors (background and foreground are inverted in the pointerover states, we should follow what the system does IMO, and fix CalculatorButtons instead of bringing this behavior to other buttons).

Reply:
Good point! Ty for taking a look.